server/api/person/person.routes.js

```diff
@@ -46,4 +46,8 @@ module.exports = function (server) {
       ]
     })
   )
+
+  server.get('/api/test/raygun', function (req, res) {
+    throw new Error()
+  })
```

Comment:
Note that this is hear for testing, and will be removed once we're happy that everything works as it should.